jobs/release.go

```diff
@@ -0,0 +1,21 @@
+package jobs
```

Comment:
Making this registration happen in the `release` package may be a way of breaking a dependency such that the types don't have to go in the top level -- and would seem more sensible to me either way. Just a thought.